functions/background/index.js

```diff
@@ -15,7 +15,7 @@
 'use strict';
 
 // [START functions_background_promise]
-const requestPromiseNative = require('request-promise-native');
+const {request} = require('gaxios');
```

Comment:
Nit: does the leading G in `gaxios` stand for Google here?

(If so, perhaps it should be moved to the `GoogleCloudPlatform` org instead of a personal repo.)

Reply:
This is a Google owned module :) It lives in the `googleapis` org:
https://github.com/googleapis/gaxios/

Reply:
Ah - I was going off of [the `npm` entry](https://www.npmjs.com/package/gaxios). Maybe we should update that? 🙂

Reply:
Good catch!  https://github.com/googleapis/gaxios/pull/239